templates/CRM/Contribute/Form/Contribution/OnBehalfOf.tpl

```diff
@@ -54,138 +54,149 @@
     selectCreateOrg(orgOption, false);
 
     if (is_for_organization.length) {
+      showHideOnBehalfOfBlock();
+
+      is_for_organization.on('change', function() {
+        showHideOnBehalfOfBlock();
+      });
+    }
```

Comment:
@mattwire I think this is wrong when i look at https://github.com/civicrm/civicrm-core/pull/17672/files?w=1 it looks like the on change is within the .length if when it should be like 

```js
    if (is_for_organization.length) {
      showHideOnBehalfOfBlock();
    }

    is_for_organization.on('change', function() {
      showHideOnBehalfOfBlock();
    });
```

Reply:
Ignore me this makes sense as length doesn't care if the element is visually hidden works otherwise adding merge on pass